apps/precinct-scanner/src/App.css

```diff
@@ -76,10 +76,9 @@ html {
 @media print {
   html {
     background: #ffffff;
-    color: #000000;
 
     /* Adjust printed ballot font-size */
-    font-size: 14px !important; /* stylelint-disable-line declaration-no-important */
+    font-size: 16px !important; /* stylelint-disable-line declaration-no-important */
```

Comment:
The changes to this file were to line up with bmd's css file as I noticed some differences in the printed polls report, but I don't have strong feelings on what the text sizes should be.